vms/registration/forms.py

```diff
@@ -1,13 +1,31 @@
+import re
+
 # Django
 from django import forms
 from django.contrib.auth.models import User
-
+from django.core.exceptions import ValidationError
 
 class UserForm(forms.ModelForm):
     # password not visible when user types it out
     password = forms.CharField(widget=forms.PasswordInput())
     confirm_password = forms.CharField(widget=forms.PasswordInput())
 
+    def clean_password(self):
+        password = self.cleaned_data['password']
+        x = r'^(?=.*?[a-z])'
+        y = '[~!@#$%^&*()_+{}":;\']+$'
+        z = r'^(?=.*?[0-9])'
+        digit = re.match(z, password)
+        special_char = set(y).intersection(password)
+        uppercase = re.match(x, password)
```

Comment:
It's checking for lower case. Isn't?

Reply:
if its checking lowercase, shouln't this be named lowercase?